doajtest/unit/test_prune_marvel.py

```diff
@@ -0,0 +1,38 @@
+import datetime, calendar
+from dateutil.relativedelta import relativedelta
+from freezegun import freeze_time
+
+from doajtest.helpers import DoajTestCase
+from portality.scripts.prune_marvel import generate_delete_pattern
+
+
+class TestFeed(DoajTestCase):
```

Comment:
Is this classname as intended?

Reply:
Yes, I have decided to name all my tests that!
(fixed ta)